.clang-format

```diff
@@ -0,0 +1,24 @@
+BasedOnStyle: LLVM
+BraceWrapping:
+  SplitEmptyFunction: false
+  SplitEmptyRecord: false
+IndentWidth: 3
+ColumnLimit: 120
```

Comment:
120 is way too small. I'd like to see something more on the order of 200

Reply:
For context, here are the default settings in the predefined styles:

```
ColumnLimit:     80  (chromium) 
ColumnLimit:     79  (gnu) 
ColumnLimit:     80  (google) 
ColumnLimit:     120  (leap_proposed) 
ColumnLimit:     80  (llvm) 
ColumnLimit:     120  (microsoft) 
ColumnLimit:     80  (mozilla) 
ColumnLimit:     0  (webkit) 
```